pkg/rbeconfigsgen/options.go

```diff
@@ -79,6 +79,9 @@ type Options struct {
 	// command to generate C++ configs inside the toolchain container. Only one of CppGenEnv or
 	// CppGenEnvJSON can be specified.
 	CppGenEnvJSON string
+	// CPPToolchainTarget is the toolchain to be used by the cpp configs. It overrides the default
```

Comment:
Also, I suggest creating a new variable called `CPPToolchainTarget` at the top level in the Options struct to align with the fact that all command line variables translate to Option fields.

This way, `CppToolchainTarget` inside `PlatformParams` is only populated once we know the full target path based on the config dir (if specified)